database/migrations/000101_remove_entities_tables.up.sql

```diff
@@ -20,4 +20,72 @@ DROP TABLE IF EXISTS repositories;
 DROP TABLE IF EXISTS artifacts;
 DROP TABLE IF EXISTS pull_requests;
 
+CREATE VIEW repositories AS
```

Comment:
I'm not as familiar with views and especially replacing tables with views, so maybe this is a dumb question - but how do indexes work?

Also comparing the columns, would we miss something by not having the reminder_last_sent and updated_at columns?

Reply:
Well, we don´t really care as much about the `updated_at` since we care now about properties. I do wonder how `reminder_last_sent` would work... would that need to be then part of the central table? maybe it needs to be.